### ISSUE TITLE
Documentation:  Missing `:` character in  manifest sample 

### ISSUE THREAD

**Issue by jvmlet** [spring-cloud/spring-cloud-dataflow-server-cloudfoundry#374]
Hi
Please add missing  `:` to the manifest sample in `2.3. Deploying using a Manifest` section :

````yaml
 SPRING_APPLICATION_JSON  {"maven": { "remote-repositories": { "repo1": { "url": "https://repo.spring.io/libs-release"} } } }
````

**Comment by sabbyanandan** [comment]
Please feel free to submit a PR where you find appropriate. We welcome contributions!

**Comment by sabbyanandan** [comment]
Oh btw, we fixed it on the 1.3.x line via https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/commit/6aa7606d57ea1b5da233e3bbf5491377cf876354, but it hasn't been backported yet - we will fix it.
